media/base/pipeline.h

```diff
@@ -268,6 +268,9 @@ class MEDIA_EXPORT Pipeline {
   // Return SetBoundsCB if SbPlayer is used for rendering.
   using SetBoundsCB = base::OnceCallback<bool(int x, int y, int width, int height)>;
   virtual SetBoundsCB GetSetBoundsCB() = 0;
+  virtual std::vector<std::string> GetAudioConnectors() const {
```

Comment:
Could this default to `=0`, so once this is upstreamed, it is not required for other pipelines to implement it.

Reply:
nit: Add comment for `GetAudioConnectors` as well.